### ISSUE TITLE
[Feature Request] Add ngx.shared.DICT.decr method

### ISSUE THREAD

**Issue by sdurrheimer** [openresty/lua-nginx-module#826]
Hi there,

Would it be possible to add the `ngx.shared.DICT.decr` method given that there is a `ngx.shared.DICT.incr` method ?

**Comment by agentzh** [comment]
@sdurrheimer Yes. Actually I've been thinking about that. I hope unlike `incr`, `decr` does not accept negative values and cannot decrement to below zero :) Patches welcome!
